libraries/GCS_MAVLink/GCS_FTP.cpp

```diff
@@ -505,7 +483,27 @@ void GCS_MAVLINK::ftp_worker(void) {
                             break;
                         }
 
-                        const uint32_t transfer_size = 100;
+                        /*
+                          calculate a burst delay so that FTP burst
```

Comment:
Would've been nice to have this as a separate PR.

Looks OK, but I think it would need flight testing on SiK radios.  We don't want reports of "no link" from users to be the first we know that ftp'ing a file in-flight breaks links.

Reply:
the change needs to be against this PR, as the sends have to be from this thread to be effective, and we don't want to sleep on main thread